keyboards/wanten/info.json

```diff
@@ -0,0 +1,154 @@
+{
+    "keyboard_name": "wanten", 
+    "url": "", 
+    "maintainer": "qpockets", 
+    "width": 12.75, 
+    "height": 4, 
```

Comment:
```suggestion
```

Reply:
i didn't know this part wasn't necessary anymore. thanks!